src/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -126,6 +126,7 @@ public static partial class Queryable
         public static TSource SingleOrDefault<TSource>(this System.Linq.IQueryable<TSource> source) { throw null; }
         public static TSource SingleOrDefault<TSource>(this System.Linq.IQueryable<TSource> source, System.Linq.Expressions.Expression<System.Func<TSource, bool>> predicate) { throw null; }
         public static System.Linq.IQueryable<TSource> Skip<TSource>(this System.Linq.IQueryable<TSource> source, int count) { throw null; }
+        public static System.Linq.IQueryable<TSource> SkipLast<TSource>(this System.Linq.IQueryable<TSource> source, int count) { throw null; }
```

Comment:
Note that I left out all versioning concerns in my commit. This will need to be `#if netcoreapp11`